apollo-compiler/src/main/kotlin/com/apollographql/android/compiler/SchemaTypeSpecBuilder.kt

```diff
@@ -40,10 +41,7 @@ class SchemaTypeSpecBuilder(
             it
                 .withValueInitConstructor()
                 .withCreatorImplementation()
-                .withFactoryImplementation()
-                .toBuilder()
-                .addType(mapper)
-                .build()
+                .withFactoryImplementation(exclude = listOf(mapper.name))
```

Comment:
making sure that was intended,  the `toBuilder().build()` calls are no longer needed?